tooling/nargo_fmt/src/formatter/doc_comments.rs

```diff
@@ -36,6 +36,48 @@ impl<'a> Formatter<'a> {
             }
         }
     }
+
+    /// Formats outer doc comments, turning them into regular comments if they start with "Safety:"
+    pub(super) fn format_outer_doc_comments_checking_safety(&mut self) {
+        self.skip_comments_and_whitespace();
+
+        let is_safety = match &self.token {
+            Token::LineComment(comment, Some(DocStyle::Outer))
+            | Token::BlockComment(comment, Some(DocStyle::Outer)) => {
+                comment.trim().to_lowercase().starts_with("safety:")
```

Comment:
Nit: this currently matches/preserves `sAfEtY` or `sAFeTY`. It may make sense to normalize to `[sS]afety` when formatting.

Reply:
Just to be clear, this code is for when there's `/// Safety` and it's changed into `// Safety`. That means that there's no specific code to turn `// sAfEtY` into `// Safety`. So maybe both should be done at the same time... but I don't feel strongly about this.

Reply:
All good :+1: